client/components/global-notices-v2/notice.tsx

```diff
@@ -0,0 +1,44 @@
+import { Snackbar } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { store as noticesStore } from '@wordpress/notices';
+import { useCallback, useEffect, useRef } from 'react';
+
+const Notice = ( { content, ...args }: { content: React.ReactNode; [ key: string ]: any } ) => {
+	const dismissTimeoutRef = useRef< null | ReturnType< typeof setTimeout > >( null );
+
+	const { removeNotice } = useDispatch( noticesStore );
+
+	const timeoutDuration = 6000;
```

Comment:
Do we absolutely need to implement a new custom timer solution? Could we use the 10 seconds dismiss timer that comes with the `Snackbar` component instead? It would avoid introducing a different behaviour from the original component for the time being.

Later on, we could look at modifying the `Snackbar` component directly to allow for different timer durations